### PR TITLE
Add release schedule to versioning policy

### PR DESCRIPTION
**Pull request by waldirmontoya25**
## Hey, I just made a Pull Request!

I concretely described what is defined in the following code.

https://github.com/backstage/backstage/blob/61ee43ad5ba31d6ac7d6810c727504d654cf8aab/scripts/generate-merge-message.js#L44-L54
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/overview/versioning-policy.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ together. Think of it as a toolbox that comes with batteries included, but you
 can always add more plugins and libraries from the open source ecosystem as well
 as build your own.
 
+## Release Schedule
+
+Backstage releases occur on a monthly basis, specifically on the second Tuesday of each month. The first release took place in March 2022.
```
Comment (kuangp) [inline docs/overview/versioning-policy.md:28]: hm don't releases go out on the third Tuesday? eg. A release came out today (7/18)
```suggestion
Backstage releases occur on a monthly basis, specifically on the third Tuesday of each month. The first release took place in March 2022.
```

Comment (waldirmontoya25) [reply docs/overview/versioning-policy.md:28]: hehe yess, thanks for the heads up.

**Comment by waldirmontoya25** [comment]
> Hi @waldirmontoya25, should this maybe be part of the "Main Release Line" section instead of it's own section? https://backstage.io/docs/overview/versioning-policy#main-release-line

Makes sense, let me change it.

File: docs/overview/versioning-policy.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ their own release cadence and versioning policy.
 
 ## Main Release Line
 
-Release cadence: Monthly
+Release cadence: Monthly, specifically on the third Tuesday of each month. The first release took place in March 2022.
```
Comment (Rugvip) [inline docs/overview/versioning-policy.md:35]: It's currently the Tuesday before the third Wednesday of each month :sweat_smile: 

The reason for that is that we wanted it to be just before the community session.

**Review by Rugvip — CHANGES_REQUESTED** [review]
:+1:

File: docs/overview/versioning-policy.md (hunk 2 above)
Comment (waldirmontoya25) [reply docs/overview/versioning-policy.md]: Done! Thanks @Rugvip

**Review by jamieklassen — APPROVED** [review]

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**